pipeline_steps/common.groovy

```diff
@@ -21,6 +21,9 @@ def install_ansible(){
     cp -r /usr/lib64/python2.6/site-packages/selinux .venv/lib64/python2.7/site-packages/ ||:
     source .venv/bin/activate
 
+    # UG-613 change TMPDIR to directory with more space
+    export PREV_TMPDIR=\$TMPDIR
+    export TMPDIR="/var/tmp"
```

Comment:
/var/ is already ~4.5G larger than the partition we're moving from, and `/var/tmp/ ` exists and has the right perms. I'd be concerned that a possible later Jenkins update or puppet run may affect customization to the server and surprise us later.